kernel/kernel-api/src/main/java/io/delta/kernel/internal/metrics/MetricsReportSerializers.java

```diff
@@ -59,7 +71,14 @@ public static String serializeTransactionReport(TransactionReport transactionRep
       new ObjectMapper()
           .registerModule(new Jdk8Module()) // To support Optional
           .registerModule( // Serialize Exception using toString()
-              new SimpleModule().addSerializer(Exception.class, new ToStringSerializer()));
+              new SimpleModule().addSerializer(Exception.class, new ToStringSerializer()))
+          .registerModule( // Serialize StructType using toString
+              // TODO Decide how we want to serialize StructType
+              // We can serialize as a JSON string (more verbose) or using toString (less verbose)
+              // Note - JSON string isn't much more parsable? no fixed schema
```

Comment:
Great question. I think .toString is fine? Maybe ask @nicklan -- any quick decision here is fine with me.

Reply:
I think, given that this is most likely to land in a log, let's go with `toString` to not bloat logs more than we need to

Reply:
SGTM